qemu/tests/cfg/virtio_mem_numa_basic.cfg

```diff
@@ -0,0 +1,41 @@
+- virtio_mem_numa_basic:
+    only Linux
+    only Host_RHEL.m9
+    only RHEL.9
+    no s390x
+    type = virtio_mem_numa_basic
+    virt_test_type = qemu
+    login_timeout = 240
+    threshold = 0.025
+    smp = 8
+    aarch64:
+        vcpu_maxcpus = ${smp}
+    vcpu_maxcpus = 16
+    slots_mem = 20
+    maxmem_mem = 80G
+    mem = 4096
+    mem_devs = 'mem0 mem1 vmem0 vmem1'
+    vm_memdev_model = "virtio-mem"
```

Comment:
Does it seem those 4 memory devices all use `virtio-mem`?

Reply:
No, the first two don't use any virtio-mem device

Reply:
OK, since `use_mem` for `mem0 mem1` are no, this line confuses users. How about `vm_memdev_model_vmem0` `vm_memdev_model_vmem1`.

Reply:
Changed now, thanks!

Reply:
Updated now, thanks!